components/devtools/actors/inspector/accessibility.rs

```diff
@@ -83,7 +83,7 @@ impl Actor for AccessibilityActor {
             "bootstrap" => {
                 let msg = BootstrapReply {
                     from: self.name(),
-                    state: BootstrapState { enabled: false },
+                    state: BootstrapState { enabled: true },
```

Comment:
Oh this was accidental we don't need this bit.

Reply:
removed in 6052016883c7986fa7c0d97a4ff1a5af6e256253